docs/README.md

```diff
@@ -1,123 +1,55 @@
+Frontity comes with a number of useful packages for routing, API source, and so on. You can learn all about each of them in the section entitled [**Frontity Packages**](api-reference-1/README.md).
 
-If **WordPress** is great and **React** is too, then why not **combine** the two? Especially if you want to build a CMS-powered site with modern web development tools.
+The included themes, namely Mars and TwentyTwenty, are dealt with separately from the other packages under the [**Frontity Themes**](frontity-themes/README.md) section.
 
-We believe this JavaScript-based approach is gaining traction in the WordPress ecosystem, so there’s no better time to start getting familiar with it.
+## **Guides**
 
-## **How does Frontity work?**
+This is where you'll find an ever-growing series of [guides](guides/README.md). Some will help you in working with Frontity. Others will give you ideas for projects and provide you with the help you need to guide you from first steps to fully working project. This is where you can really have fun with Frontity and work toward your stretch goals.
 
-With Frontity you still use your WordPress dashboard to edit and manage your content in exactly the same way that you are accustomed to. As you make changes content is automatically updated in your Frontity site, just as it is when using a traditional WordPress theme.
+## **Contributing**
 
-Frontity apps require a Node.js server to run on. This runs in tandem with the WordPress site which is now relegated to providing content to the frontend that is based on Frontity.
+Frontity is an open source project and we welcome contributions from the community. The [**Contributing**](contributing/README.md) section of the documentation will guide you whether you want to make a single contribution or whether you intend to become a fully fledged contributor to the Frontity project. We can't wait to see where you take Frontity in the future.
 
-* Frontity requests content from the WordPress REST-API and uses it to generate the final HTML that is displayed in the browser.
-* Frontity is also capable of generating AMP pages using the same React code and CSS.
+## **And that's not all**
 
-![](.gitbook/assets/frontity-architecture%20%281%29.png)
+There's yet more to the Frontity documentation.
 
-**Why a different Node.js server?**
+[Frontity's WordPress Plugins](frontity-plugins/README.md) are plugins built by the team here at Frontity that you install and activate in your WordPress site and that will enhance your Frontity site. 
 
-React is a JavaScript library. In order to generate HTML for site visitors or for Google AMP, the server needs to be able to run JavaScript as well.
+[Key Concepts](key-concepts/README.md) will help you with some foundational knowledge that will help you when working with Frontity. Here we even provide you with introductions to JavaScript/ES6 and React, just in case you need them.
```

Comment:
this link `key-concepts/README.md` will not work until you rename this section

Reply:
Oh yes, good catch - I haven't figured out how to rename a directory on GitHub yet. I may need to create a new directory and then move the files to it.

Reply:
Aren't you working locally with the files of your fork?

Reply:
No, I'm working on GitHub - but now wish I'd pulled it to local before I started! 😄

Reply:
You can do it anytime. Notice https://github.com/mburridge/gitbook-docs how you have a branch `mburridge:Updates-to-Getting-Started-section` in [your fork](https://github.com/mburridge/gitbook-docs) so any updates of this branch will be reflected in this PR

Reply:
Changed link back to resources/README.md so this doesn't hold things up. We can change the directory name separately later on.

Reply:
The change is not reflected in this PR

Reply:
❓❓❓
https://github.com/frontity/gitbook-docs/pull/32/commits/592e2e750f8bdd23a0b26b481b4a368a196efbb8